llmc/dataloader.h

```diff
@@ -236,16 +237,17 @@ void evalloader_reset(EvalLoader *loader) {
     // For example if there are N examples in the file and 4 processes,
     // then process 0 should start at 0, process 1 at N/4, process 2 at N/2, etc.
     // determine how much work there is for all processes
-    int examples_per_process = CEIL_DIV(loader->num_examples, loader->num_processes);
-    int can_fit_examples = loader->B / ASSUMED_NUM_COMPLETIONS;
-    loader->num_batches = CEIL_DIV(examples_per_process, can_fit_examples);
+    int examples_per_process = FLOOR_DIV(loader->num_examples, loader->num_processes);
```

Comment:
isn't FLOOR_DIV just a divide / ?
what is gained by macro

Reply:
Unnecessary. I updated the source.